sdk/tvl-usd.js

```diff
@@ -0,0 +1,138 @@
+const debug = require("debug")("opentvl:tvl-usd");
+const abi = require('./abis/aggregatorV3.json')
+const sdk = require('.');
+const fetch = require('node-fetch');
+const { getPriceFeeds } = require('./lib/chainlink');
+const wait = (milliseconds) => new Promise((resolve) => setTimeout(resolve, milliseconds));
+
+const COIN_GECKO_IDS = require('./data/coinGeckoIDs.json');
+
+let feeds = {};
+getPriceFeeds().then(res => {
+  debug("finished fetching price feed proxy addresses");
+  feeds = res;
+});
```

Comment:
this optimization is a bit implicit, the return value can be undeterminated(very hard for unit tests), and made this file stateful.

I would think of hosting the state in server.js instead and use promise to ensure the server won't start listen before those preparation work were done.

```
let chainlinkFeeds = {};

fetchFeeds().then((feeds) => {
  chainlinkFeeds = feeds;
  
  app.listen(...)
})

```